TLM/TLM/State/GlobalConfig.cs

```diff
@@ -14,7 +14,7 @@ namespace TrafficManager.State {
     public class GlobalConfig : GenericObservable<GlobalConfig> {
         public const string FILENAME = "TMPE_GlobalConfig.xml";
         public const string BACKUP_FILENAME = FILENAME + ".bak";
-        private static int LATEST_VERSION = 20;
+        private static int LATEST_VERSION = 21;
```

Comment:
I'm not sure about that change. All changes you've made are not affecting config in release build, and when you try using debug build the config is regenerated anyways

Reply:
but debug switches are no longer in `TMPE_GlobalConfig.xml` anymore.

Reply:
I suppose that is OK.

Reply:
Yes they are not, **but** normal users don't have it either so there is no point of changing config if.. config didn't change